### ISSUE TITLE
VO members should be able to re-sign the AUP at any time

### ISSUE THREAD

**Issue by maarten-litmaath** [indigo-iam/iam#736]
VO members should be able to re-sign the AUP e.g. to ensure that workflows in the next days will not run into the expiration of their AUP signature and then fail. This is particularly important for accounts owning robot certificates or tokens clients used by grid services. If #735 is fixed, at least VO admins could come to the rescue, but possibly only after some damage was already done.

The GUI needs to have a button leading to the AUP confirmation page that currently is only presented when the user tries to log in _after_ the AUP has already expired.

**Comment by enricovianello** [comment]
The new Re-sign AUP button into user's home page:

![Image](https://github.com/indigo-iam/iam/assets/1914039/6e43eca3-ece9-4cef-8ffb-152f7ffa4760)

The confirmation dialog:

![Image](https://github.com/indigo-iam/iam/assets/1914039/e8bb9291-c353-4ed7-a25f-737f132a70b6)
